src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -9210,6 +9223,10 @@ function PSCreateDirectoryOnRemoteSession
                     if ($force)
                     {{
                         Microsoft.PowerShell.Management\New-Item $createDirectoryPath -ItemType Directory -Force | Out-Null
+                        if (ExperimentalFeature.IsEnabled('PSCopyItemDirectoryAttributes'))
+                        {{
+                            (Get-Item $createDirectoryPath -Force).Attributes = $attributes
+                        }}
```

Comment:
When the remote session is an older version of PowerShell, `ExperimentalFeature.IsEnabled(PSCopyItemDirectoryAttributes)` won't work. I think you will have to decide whether to set the attribute depending on if `$attribute` is `null` or `not-null`. Currently `$attributes` is of a enum type, so I guess you cannot just pass in null when the feature is disabled.